<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eigrp</class>
 <widget class="QWidget" name="Eigrp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 170, 255);
selection-background-color: rgb(66, 66, 66);
background-color: rgb(70, 70, 70);
alternate-background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>AS Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="asNumBox">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>21</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="disableCheck">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wildBox4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wildBox1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>102</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Wildcard</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wildBox2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wildBox3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idBox3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idBox2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idBox1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>172</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Router ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idBox4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="removeNetCheck">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Remove Network</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="removeIdCheck">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Remove Router ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>62</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoCheck">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Auto-Summary</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logCheck">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Log Neghbors</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
selection-color: rgb(0, 170, 255);
selection-background-color: rgb(0, 170, 255);
alternate-background-color: rgb(0, 170, 255);
gridline-color: rgb(0, 0, 127);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>Passive Interface</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color :rgb(0, 170, 255)</string>
    </property>
    <property name="text">
     <string>Interface</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interfaceBox">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>40</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="defaultPassCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>All Interfaces</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPassBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Add </string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePassBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Variance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="varianceBox">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>238</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>asNumBox</tabstop>
  <tabstop>disableCheck</tabstop>
  <tabstop>netBox1</tabstop>
  <tabstop>netBox2</tabstop>
  <tabstop>netBox3</tabstop>
  <tabstop>netBox4</tabstop>
  <tabstop>wildBox1</tabstop>
  <tabstop>wildBox2</tabstop>
  <tabstop>wildBox3</tabstop>
  <tabstop>wildBox4</tabstop>
  <tabstop>removeNetCheck</tabstop>
  <tabstop>idBox1</tabstop>
  <tabstop>idBox2</tabstop>
  <tabstop>idBox3</tabstop>
  <tabstop>idBox4</tabstop>
  <tabstop>removeIdCheck</tabstop>
  <tabstop>varianceBox</tabstop>
  <tabstop>autoCheck</tabstop>
  <tabstop>interfaceBox</tabstop>
  <tabstop>defaultPassCheck</tabstop>
  <tabstop>addPassBtn</tabstop>
  <tabstop>removePassBtn</tabstop>
  <tabstop>logCheck</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>configBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
